File: OSSA/Lab Sheets/Lab1/exe5.go
package main

import (
	"fmt"
	"os"
	"time"
)

func main() {
	fmt.Printf("Parent PID: %d\n", os.Getpid())
	go func() {
		fmt.Printf("Child PID: %d\n", os.Getpid())
		fmt.Printf("Child's Parent PID: %d\n", os.Getppid())
	}()
	time.Sleep(time.Microsecond * 20)
}

File: OSSA/Lab Sheets/Lab1/exe5.c
#include <stdio.h>
int main()
{
        int ret;

        printf("Hello World\n");
        ret = fork();
        if (ret == 0)
        {
                printf("I am Child and Return Value=%d\n", ret);
                printf("Child PID: %d\n", getpid());
                printf("Child's Parent PID: %d\n", getppid());
        }
        else
        {
                printf("I am Parent and Return Value=%d\n", ret);
                printf("Parent PID: %d\n", getpid());
        }
        sleep(20);
}
